.github/workflows/e2e.yml

```
@@ -83,7 +98,31 @@ jobs:
       - name: Install Playwright
         run: pnpm exec playwright install --with-deps
 
-      - name: Run E2E Tests
+      - name: Azure login
+        if: inputs.AZURE_CLIENT_ID != '' && inputs.AZURE_TENANT_ID != '' && inputs.AZURE_SUBSCRIPTION_ID != ''
```

actionlint (1.7.4)

property "azure_client_id" is not defined in object type {}

(expression)